src/storage_manifests.py

```diff
@@ -142,3 +137,23 @@ def evaluate(self) -> Optional[str]:
             if not self.config.get(prop):
                 return f"Storage manifests waiting for definition of {prop}"
         return None
+
+class UpdateControllerPlugin(Patch):
```

Comment:
Human Linter:  Needs 2 lines

```suggestion


class UpdateControllerPlugin(Patch):
```

Reply:
thanks!